content/courses/trigonometry/identities/law-of-sines/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Law of Sines
+decription: Law of Sines!
+index: 4
+type: lesson
+
+---
+
+## LAW OF SINES
+
+
+The Law of Sines is one of the first most important relationships
+between trigonometric functions and geometry.
+
+In [Functions](../../functions/exploration/index.md), we learned about trigonometry in right triangles. Now, let's look at a triangle with unknown angles:
```

Comment:
ah, links are not "replaced" during build. Should be `/learn/trigonometry/functions/exploration/`